src/index.js

```diff
@@ -323,11 +323,16 @@ module.exports = (
         assets[filename + '.map'] = { source: JSON.stringify(map), permissions: defaultPermissions };
         map = undefined;
       }
-      code = `const { readFileSync, writeFileSync } = require('fs'), { Script } = require('vm'), { wrap } = require('module');\n` +
+      code =
+        `if (process.pkg) {\n` +
```

Comment:
How are you setting `process.pkg`?

Reply:
`pkg` sets this before running any user code. So one can check the mode

Reply:
Ahh, I see :)

Reply:
😄  https://github.com/zeit/pkg